### PR TITLE
Updated currency formatting

### PR DESCRIPTION
**Pull request by FrederikBolding**
## Description

- Updated currency formatting to include commas in larger numbers
- Used currency formatting in WalletBreakdown
- Used fiat setting for WalletBreakdown

### CODE REVIEW

File: common/v2/components/Currency.tsx [hunk 1]
```diff
@@ -15,12 +15,25 @@ interface Props {
...
   icon?: boolean;
   prefix?: boolean;
+  useTypography?: boolean;
 }
 
-function Currency({ amount, symbol, decimals = 5, icon = false, prefix = false, ...props }: Props) {
+function ConditionalTypography({ condition, children }: { condition: boolean; children?: any }) {
+  return condition ? <Typography>{children}</Typography> : children;
+}
+
```
Comment (vilsbole) [inline common/v2/components/Currency.tsx]: Adding a prop `useTypography` creates a strong dependance between the 2 components (it means the caller knows the implementation of the callee).  

Ideally we want every displayed text to be wrapped with a `Typography` component which would handle all our font scaling. The component scales poorly so it's wrapped with `v2/components/Typography`.

Prefer making the desired style modifications / flags to `v2/components/Typography`.

Comment (FrederikBolding) [reply common/v2/components/Currency.tsx]: So removing `PanelFigureValue` and making `Typography` and thereby `Currency` support props that allow for similar styling?

Comment (vilsbole) [reply common/v2/components/Currency.tsx]: Yes that is the right direction!

Comment (FrederikBolding) [reply common/v2/components/Currency.tsx]: Pushed changes, let me know what you think!

**Pull request merged**